js/app.js

```diff
@@ -363,16 +349,17 @@
 				});
 			}
 		},
+		addParticipantToCall: function(token, participant) {
+			return this.signaling.addParticipantToCall(token, participant);
```

Comment:
Yeah, the database has a (persistent) list of users invited to a room. This is queried by the signaling when a user wants to join. The list of currently connected users in a room is kept by the signaling server (non-persistent).

Once a user is no longer invited (i.e. gets kicked), a notification must be sent from the db to the signaling to close any active connections of that user (to the room she is no longer invited to).